### ISSUE TITLE
[Android] Return animation seems to offset to the left

### ISSUE THREAD

**Issue by SamitJsn** [IjzerenHein/react-navigation-shared-element#99]
On iOS the sharedElement seems to perform well and return to its exact position. However, on Android, it returns to its same y position but has this slight adjustment of it's x position as it returns to the edge of the screen then to it's original position

This is the shared element that I create, it is rendered as part of of a flatList
```
            <SharedElement id={item.headerText}>
                <View style={styles.item}>
                    {getImage(item.headerText)}
                    <View style={styles.textContainer}>
                        <Text style={styles.headerText}>{item.headerText}</Text>
                        <Text style={styles.descriptionText}>{item.description}</Text>
                    </View>
                </View>
            </SharedElement>
```

You can see below that the element snaps to the edge of the screen then corrects itself
![Screen-Recording-2020-08-29-at-1](https://user-images.githubusercontent.com/29716622/91626202-71ddb700-e9f0-11ea-93fc-0c945baa75c8.gif)

**Comment by JaapWeijland** [comment]
I experience the same thing in iOS (react-navigation@5)

**Comment by akinyemi-clane** [comment]
Same thing with react-navigation-v5

**Comment by ajinkabeer** [comment]
Same with react-navigation-v5

**Comment by JaapWeijland** [comment]
Using `SafeAreaView` from `react-native-safe-area-context` instead of `react-native` seems to fix this issue.

**Comment by jp928** [comment]
Same on iOS

**Comment by iDuuck** [comment]
Please check the other issue, where I posted a (possible) fix for this issue: https://github.com/IjzerenHein/react-navigation-shared-element/issues/106#issuecomment-723493372

**Comment by jwoodmansey** [comment]
> Using `SafeAreaView` from `react-native-safe-area-context` instead of `react-native` seems to fix this issue.

Even using `react-native-safe-area-context` was causing issues for me, not exactly sure why. I've got this working by implementing my own safe areas on the screens with problematic transitions.

**Comment by KawaljeetCuelogic** [comment]
@jwoodmansey Please have a look here https://github.com/IjzerenHein/react-navigation-shared-element/issues/147#issuecomment-788140353 . What according to you might be causing problems here?

**Comment by IjzerenHein** [comment]
Hi! This was a problem in the Native Android code. It has been fixed in `v0.8.0` of `react-native-shared-element`
https://github.com/IjzerenHein/react-native-shared-element/releases

Please upgrade your native code and this problem should be fixed.

**Comment by IjzerenHein** [comment]
> > Using `SafeAreaView` from `react-native-safe-area-context` instead of `react-native` seems to fix this issue.
> 
> Even using `react-native-safe-area-context` was causing issues for me, not exactly sure why. I've got this working by implementing my own safe areas on the screens with problematic transitions.

Please read the comments about `SafeAreaView` in this thread: https://github.com/IjzerenHein/react-navigation-shared-element/issues/187#issuecomment-904728025
